src/ui/castlekeysmouse.pas

```diff
@@ -767,6 +771,10 @@ function StrToKey(const S: string; const DefaultKey: TKey): TKey;
 
     { Textual description of this event. }
     function ToString: string;
+    { Character corresponding to EventType = itKey
+      Returns #0 if the event was not a keyboard event or keypress event
+      wasn't a simple keyboard event (e.g. a Cyrillic or Arabic character) }
+    function KeyCharacter: char;
```

Comment:
Make sure to end sentences with a dot. It looks better, and it is also used by https://github.com/pasdoc/pasdoc/wiki/AutoAbstractOption for the 1st sentence.

I propose to change

```
{ Character corresponding to EventType = itKey
  Returns #0 if the event was not a keyboard event or keypress event
  wasn't a simple keyboard event (e.g. a Cyrillic or Arabic character) }
```

to this:

```
{ Character corresponding to EventType = itKey.
  Returns #0 if the event was not a keyboard event or it cannot be
  represented as a simple 8-bit character (e.g. it is a Cyrillic or Arabic character,
  or it is a special key like "up arrow"). }
```

Reply:
Done.